app/components/shared/previews/notification_component_preview.rb

```diff
@@ -0,0 +1,21 @@
+class Shared::Previews::NotificationComponentPreview < ViewComponent::Preview
+  layout "design_system"
+
+  def self.component_name
+    "notification"
```

Comment:
Alternatively,

```ruby
compent_class.to_s.underscore.humanize.split("/").second.split(" ").first
# or if we get rid of the 'shared' folder:
compent_class.to_s.underscore.humanize.split(" ").first
```

Reply:
But that's probably premature optimization

Reply:
("component" is gsubbed in `app/views/design_system/_nav_side.html.slim` but could equally well do so here)

Reply:
no i think its a simple change that removes some hardcodedness, and thats usually a good thing